config.go

```diff
@@ -352,12 +352,7 @@ type Config struct {
 
 	DebugLevel string `short:"d" long:"debuglevel" description:"Logging level for all subsystems {trace, debug, info, warn, error, critical} -- You may also specify <global-level>,<subsystem>=<level>,<subsystem2>=<level>,... to set the log level for individual subsystems -- Use show to list available subsystems"`
 
-	CPUProfile string `long:"cpuprofile" description:"Write CPU profile to the specified file"`
```

Comment:
this change just broke my local node - need to use the hidden flag instead of removing them, also should mention the change in the release notes so operators know these configs are now hidden and will be removed.

Reply:
sorry 😢